RZUtils/Categories/UIView/UIView+RZAutoLayoutHelpers.m

```diff
@@ -222,6 +222,106 @@ - (NSLayoutConstraint*)rz_pinnedCenterYConstraint
     return constraint;
 }
 
+- (NSArray *)rz_bottomToTopConstraints
+{
+    if ( self.superview == nil ) {
+        return nil;
+    }
+
+    NSMutableArray *constraints = [NSMutableArray array];
+    for ( NSLayoutConstraint *c in self.superview.constraints ) {
+        if ((( c.firstItem == self && [c.secondItem superview] == self.superview ) && c.firstAttribute == NSLayoutAttributeBottom && c.secondAttribute == NSLayoutAttributeTop ) ||
+            (( c.secondItem == self && [c.firstItem superview] == self.superview ) && c.secondAttribute == NSLayoutAttributeBottom && c.firstAttribute == NSLayoutAttributeTop )) {
```

Comment:
Can you show me what space I'm missing when you have a minute? Having a brain fart.